packages/wrangler/src/sites.ts

```diff
@@ -118,7 +118,7 @@ function pluralise(count: number) {
  * @returns a promise for an object mapping the relative paths of the assets to the key of that
  * asset in the KV namespace.
  */
-export async function syncAssets(
+export async function syncSitesAssets(
```

Comment:
To match naming conventions:

```suggestion
export async function syncLegacyAssets(
```

The fact this is located in the `sites.ts` module is enough to make the connection